web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -1841,6 +1841,17 @@
                   : "";
             return angular.isFunction(this[fnName]) ? this[fnName]() : false;
           },
+          canViewMetadataHistory: function () {
+            var profile = gnConfig["metadata.history.accesslevel"] || "Editor",
+              fnName =
+                profile !== ""
+                  ? "is" + profile[0].toUpperCase() + profile.substring(1) + "OrMore"
+                  : "";
+            if (profile === "RegisteredUser") {
```

Comment:
Is this check required because there is no `RegisteredUserOrMore` function defined I guess?

Reply:
There is no RegisteredUserOrMore function and this is why I am adding this condition. There is a case when the user is totally not logged in, like the was access by some random internet visitor